2-mandatory/1-weather-report.js

```diff
@@ -12,7 +12,12 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let cityNameAndDegree = [];
+    
+    for(let temp of cities){
+        cityNameAndDegree.push(`The temperature in ${temp} is ${temperatureService(temp)} degrees`)
+    }
```

Comment:
This works well. "city" would be a better variable name than "temp" here.

Reply:
Done